samples/apps/copilot-chat-app/webapi/Services/AzureContentModerator.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI;
+using SemanticKernel.Service.CopilotChat.Options;
+
+namespace SemanticKernel.Service.Services;
+
+public record AnalysisResult(
+    [property: JsonPropertyName("category")] string Category,
+    [property: JsonPropertyName("riskLevel")] short RiskLevel
+);
+
+public record ImageContent([property: JsonPropertyName("content")] string Content);
```

Comment:
I'm curious. Why make this a record? Couldn't we just let it be a string? Or does that have something to do with how data is sent back to us from the service?

Reply:
it's required by the service as the request schema